.devcontainer/devcontainer.json

```
@@ -4,6 +4,7 @@
   "features": {
     "ghcr.io/devcontainers/features/node:1": {
       "version": "22.17.0",
+      "pnpmVersion": "^10.13.1",
```

We should define the exact version here. Otherwise pnpm will complain about mismatches with package manager version in `package.json`.

```suggestion
      "pnpmVersion": "10.13.1",
```